build.gradle

```diff
@@ -91,12 +91,12 @@ repositories {
 ext {
     junitJupiterVersion = '5.5.2'
     junitVintageVersion = '5.5.2'
-    powermockVersion = '2.0.0-beta.5'
+    powermockVersion = '2.0.7'
     reformLogging = '5.1.0'
     appInsightsVersion = '2.4.1'
-    swagger2Version = '2.10.0'
+    swagger2Version = '3.0.0'
```

Comment:
Fairly sure Swagger 3 has breaking changes that would require config updates. Can you please check whether Swagger is still working?